### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by Katbow**
This is an overhaul of both the README and CONTRIBUTING guides. I changed the README to the most basic rules in the hopes of making it less intimidating and easier to read. I moved the detailed instructions to the CONTRIBUTING guide. 

If there are still some parts of the guide that are confusing, we can possibly add images in a future pull request. 

I'll merge after getting some feedback & thumbs up from others.

### CODE REVIEW

**Comment by tas12** [comment]
Hey @Katbow looks good. For step 1: I think when a new member needs to open an issue to request a problem you could include that bit about what they should name the issue. Then it's easier to identify issues that are new problem requests.

File: README.md [hunk 1]
```diff
@@ -1,49 +1,16 @@
...
-**Youtube introduction to GitHub (30 minutes)**
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/codingforeveryone/js?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-https://www.youtube.com/playlist?list=PLfdtiltiRHWFEbt9V04NrbmksLV4Pdf3j
+# Using this repository
 
-**GitHub help on 'Fork a repo'**
-https://help.github.com/articles/fork-a-repo/
+This repository is for becoming familiar with GitHub by completing JavaScript problems.
+There is a process to follow for contributing. The basics are outlined below, and full details/help can be found in [the CONTRIBUTING.md file](CONTRIBUTING.md).
```
Comment (andrewMacmurray) [inline README.md]: This all looks great, maybe add something here like "Join our chat room on gitter, we'd love to hear from you!", just to make it very clear this is a friendly and sociable repo

**Comment by tu6619** [comment]
Looking good @Katbow! What about adding something to the line "Keep the problem simple!"? Maybe something like "Have a look at 8kyu and 7kyu Katas on Codewars for inspiration!"

**Comment by andrewMacmurray** [comment]
@tu6619 I think that's a good idea

**Comment by tas12** [comment]
Do you guys think I should change new member problem 9? I would have done something simpler like calculating the area of a shape but didn't want it to be too boring. I guess that's not the point though. Let me know your thoughts

**Comment by Katbow** [comment]
Okay great suggestions from everyone! I think what I will do is merge as is, then put these suggestions as an issue. I'll wait to see if there's anyone else who wants to leave feedback for it, then merge tomorrow.

**Comment by philb56** [comment]
@Katbow - new instructions looking good - I have just gone through the 'new member' process using the instructions (just before your  changes) and all good so far. I am now confused about what to do next. May be a comment or two might be helpful.

**Comment by Katbow** [comment]
@philb56 do you mean it is difficult to follow through the new instructions I posted? Or that you are unsure what to do after completing the instructions? Since this repository is for trying out/practicing with GitHub, you don't need to do anything in particular. If you would like more practice you could certainly go through the process a second time provided there is a problem available to complete. Otherwise there are [some other repositories you can contribute to](https://github.com/codingforeveryone), if you would like.

**Comment by Katbow** [comment]
Oh @tas12 I missed your last comment. I think that problem is just fine.

**Comment by Katbow** [comment]
Okay I merged and am just posting an issue with the suggestions.

**Comment by philb56** [comment]
@Katbow Definitely the latter - the instructions were fine. Thanks for the tips - I'll check out the other repos.

**Pull request merged**